doc/man3/BIO_ctrl.pod

```diff
@@ -12,8 +12,8 @@ BIO_get_info_callback, BIO_set_info_callback, bio_info_cb
 
  #include <openssl/bio.h>
 
- typedef void (*bio_info_cb)(BIO *b, int oper, const char *ptr,
-                             int arg1, long arg2, long arg3);
+ typedef void bio_info_cb(BIO *b, int oper, const char *ptr,
+                          int arg1, long arg2, long arg3);
 
  long BIO_ctrl(BIO *bp, int cmd, long larg, void *parg);
  long BIO_callback_ctrl(BIO *b, int cmd, bio_info_cb cb);
```

Comment:
this should use `bio_info_cb *cb` as a consequnece.

Reply:
Thanks.  Fixed, and I looked through the other corrected typedefs as well.